content/future.html

```diff
@@ -2,104 +2,133 @@ <h1 id="future"> Future </h1>
-  graduating. Although most of the class plans to join the workforce immediately,
-  34% of students are considering grad school in the future.
-  <br/><br/>
-
-  Even though 80% of the class has worked in California, less than half of
-  students have chosen to live in California after graduating. Although California
-  is perceived by many as the place to be for software engineers, seeing the gap
-  in the conversion rate suggests that there are many other great places for
-  software engineers such as East Coast Canada, East Coast US and Pacific North
-  West US.
+  However, only 52 of the 85 SE graduates, and a few who transferred, answered this
+  question. The only take-away is that most students had the opportunity to
+  accept a job offer prior to graduating.
+  <!-- 85 - 52  = 33 -->
 </p>
 <div class="half second">
   <span class="graph-title">Post graduation plans</span>
   <div id="post-grad"></div>
   <div class="vb"></div>
-  <span class="graph-title">Desired location</span>
+  <span class="graph-title">Relocation plans</span>
   <div id="post-location"></div>
 </div>
 
-<div class="vbx2"></div>
+<div class="vbx3"></div>
 
-<h3> Motivations </h3>
 <p class="half">
   <span class="big">
-    Good compensation and growth as an engineer were the main
-    factors affecting post-graduation plans.
+    The co-op acts as a great pipeline for hiring young talent for companies.
   </span>
-  This could explain why California was a popular choice for full-time location.
-  <br /><br />
-  Although family may be the least popular factor, most students working away
-  from home have expressed interest in working near their families in the
-  future.
+  It also enables students find a workplace they enjoy, as shown by the
+  sheer amount of student opting to return to a co-op employer rather
+  than accept an offer elsewhere.
 </p>
 <div class="half second">
-  <span class="graph-title">Factors affecting post-graduation plans</span>
-  <div id="motivation"></div>
+  <span class="graph-title">Co-op conversion</span>
+  <div id="coop-conversion"></div>
 </div>
 
-<div class="vbx2"></div>
+<div class="vbx3"></div>
 
-<h3> Debt </h3>
 <p class="half">
   <span class="big">
-    65% of students are graduating debt-free.
+    Students are very content with their post-grad plans with 52% feeling
+    they are perfect and 91% rating them a rating of 4/5 and higher.
   </span>
-  30% of students have less than $30,000 in debt. Among students who are
-  in debt, the median debt is $22,500.
+  Given the state of the world at this moment, it is reassuring to see
+  SE students being satisfied with their situations.
+</p>
+<div class="half second">
+  <span class="graph-title">Post-grad contentness</span>
+  <div id="post-contentness"></div>
+</div>
+
+<div class="vbx2"></div>
+<h3> Full-Time Compensation </h3>
+<p class="half">
+  <span class="big">
+    Compensation for the first year of full-time work for respondants will be
```

Comment:
"The compensation for the first-year of full-time work for respondents will average $284,728 CAD, with a median of $260,000."

reworded it slightly